Environments/IDisplayable.java

```diff
@@ -0,0 +1,9 @@
+package Environments;
+
+import java.awt.*;
+
+public interface IDisplayable {
```

Comment:
Why do you have two `IDisplayable` interfaces?  `Environments/IDisplayable.java` and `IDisplayable.java`